database/migrations/2023_07_16_114745_rename_address_to_google_column_from_monsters.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Not sure we really need to rename anything here, it's just adding complexity to the PR. 

If we do it, we do it in a separate PR. renaming the column has no benefit in this PR

Reply:
Removing it!